### PR TITLE
Fix #8204: Crash when tile element has no surface elements.

### LINKED ISSUE #8204: BACKTRACE e5ef41
<table>
    <tr>
        <td></td>
        <td><a href='https://openrct2.sp.backtrace.io/dashboard/openrct2/project/openrct2/group/e5ef41261de97b7fd3a21ddcd782b17d34e9399be968148c0054f6724e37ef22?_t=W3sibmFtZSI6IndvcmtmbG93In1d'><b>New unique error: [openrct2]</b></a><br /><b>1</b> occurrence(s) <b></b> since (Fri Nov 02 2018 00:52:00)<br /></td>
    </tr>
</table>

<table>
<tr><td><strong>Application</strong></td><td>openrct2.exe</td></tr>

<tr><td><strong>Attributes</strong></td><td><strong>commit:</strong> f8add7f: <em>1</em><br></td></tr>
<tr><td><strong>Classifiers</strong></td><td><strong>invalid-read</strong> </td></tr>
<tr><td><strong>Frames</strong></td><td><code>vehicle_update_measurements -> vehicle_update -> OpenRCT2::GameState::UpdateLogic -> OpenRCT2::GameState::Update -> OpenRCT2::Context::Update -> OpenRCT2::Context::RunGameLoop -> OpenRCT2::Context::RunOpenRCT2 -> NormalisedMain -> LaunchOpenRCT2</code></td></tr>
</table>

### PR DESCRIPTION
**Pull request by ZehMatt**
Simply a missing null pointer check.

### CODE REVIEW

File: src/openrct2/ride/Vehicle.cpp [hunk 1]
```diff
@@ -1814,7 +1814,7 @@ static void vehicle_update_measurements(rct_vehicle* vehicle)
 
     TileElement* tile_element = map_get_surface_element_at({ x, y });
     // If vehicle above ground.
-    if (tile_element->base_height * 8 <= z)
+    if (tile_element && tile_element->base_height * 8 <= z)
```
Comment (Gymnasiast) [inline src/openrct2/ride/Vehicle.cpp]: We normally write ` != nullptr`.
Also, could you amend the changelog?

Comment (ZehMatt) [reply src/openrct2/ride/Vehicle.cpp]: Sure

File: distribution/changelog.txt [hunk 2]
```diff
@@ -27,6 +27,8 @@
 - Fix: [#8121] Crash Renaming park with server logging enabled.
 - Fix: [#8142] Reliability of mazes and crooked houses can go below 100%.
 - Fix: [#8187] Cannot set land ownership over ride entrances or exits in sandbox mode.
+- Fix: [#8204] Crash when tile element has no surface elements.
+
```
Comment (janisozaur) [inline distribution/changelog.txt]: Unnecessary whitespace

Comment (janisozaur) [reply distribution/changelog.txt]: I would have amended this myself, but was on mobile and the editor can't remove lines there.

**Pull request merged**